src/main/java/games/strategy/engine/lobby/server/db/UserDao.java

```diff
@@ -9,10 +9,16 @@
   boolean doesUserExist(String userName);
 
   void updateUser(DBUser user, HashedPassword password);
+  
+  void updateUser(DBUser user, String password);
```

Comment:
What was the intended way for someone to know which method is the right one to use? Were the new methods meant to be temporary and the `HashedPassword` ones deleted over time?